src/steam/steam_web_api.ts

```diff
@@ -10,7 +10,7 @@ export default class SteamWebAPI {
   public static logger = new Logger('SteamWebAPI');
   public static instance = axios.create({
     baseURL: 'https://api.steampowered.com/',
-    timeout: 1000,
+    timeout: 5000,
```

Comment:
Let's make this an uppercase const at the top of the file to make it more noticeable

Reply:
could even be a config parameter with a default value of 5000

Reply:
I choose the const for now, maybe we can change it to config later